README.md

```diff
@@ -2,3 +2,45 @@
 Parse Public Cloud image names that match the SUSE naming convention
 
 API implementation to parse image names that match the SUSE Public Cloud image naming convention.
+
+## Quick Start
+
+### Install from Source
+
+```bash
+  # Clone the repo
+  $ git clone https://github.com/SUSE-Enceladus/suse-cloud-image-name-parser
+  $ cd suse-cloud-image-name-parser 
+
+  # create the virtual environment.
+  $ virtualenv dev_venv --python=python3
+
+  # activate the virtual environment for interactive bash session
+  $ source dev_venv/bin/activate
+
+  # There is 2 ways you can install
+    1. Install latest sources in developer mode
+       $ pip install -e .[dev]
+
+    2. Install latest version normally
+       $ pip install .[dev]
+```
+
```

Comment:
Should we show a sample usage here or in a USAGE.md? I did not see any documentation (other than the docstring in the module) on using the image_name api. I know most of us know how to use this but for example maybe give some guidance on the image name format and valid frameworks possibly? If you think the doc string is sufficient that is fine but someone looking at this may not know that this comment in the docstring,  "such as any of the image names reported by ``pint`` ". refers to the pint.suse.com or susepubliccloudinfo client. Just a suggestion